be/src/cache/object_cache/object_cache.cpp

```diff
@@ -40,11 +44,23 @@ Status ObjectCache::init(const ObjectCacheOptions& options) {
         LOG(ERROR) << "Unsupported cache module";
         return Status::NotSupported("unsupported block cache engine");
     }
-    RETURN_IF_ERROR(_cache_module->init());
     _initialized.store(true, std::memory_order_release);
     return Status::OK();
 }
 
+#ifdef WITH_STARCACHE
+Status ObjectCache::init(const std::shared_ptr<starcache::StarCache>& star_cache) {
+    if (_initialized.load(std::memory_order_acquire)) {
```

Comment:
Consider using `std::call_once` for more simplicity and semantics.

Reply:
I will fix it later